scripts/image-builder/config/prometheus-rpms.toml.template

```diff
@@ -0,0 +1,7 @@
+id = "prometheus-rpms"
```

Comment:
This file is probably not required since you're reusing the custom RPM script option, right?